README.md

```diff
@@ -40,9 +40,9 @@ import { BrowserModule } from '@angular/platform-browser';
 import { TreeModule } from 'ng2-tree';
 
 @NgModule({
-  declarations: [MyComponent],
+  declarations: [AppComponent],
```

Comment:
`MyComponent` is correct, cause I was describing the one that is below module definition at https://github.com/valor-software/ng2-tree#clapper-usage. It is not connected to the demo